benchmarks/vectorized_random_hue.py

```diff
@@ -51,7 +51,7 @@ class OldRandomHue(BaseImageAugmentationLayer):
 
     """
 
-    def __init__(self, factor, value_range, seed=None, **kwargs):
+    def __init__(self, factor, value_range=(0, 255), seed=None, **kwargs):
```

Comment:
please remove default values for value_range

Reply:
Done.